README.md

```diff
@@ -28,8 +28,8 @@ collections:
   - cohesity.dataprotect
 ```
 # Requirements
-- ansible version >= 2.9
```

Comment:
Isn't that a big jump from 2.9 to 8? 
Whats the latest ansible release available?

Reply:
The docs was not updated from a long time here.